helmfile.d/upstream/falcosecurity/falco/Chart.lock

```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: falcosidekick
   repository: https://falcosecurity.github.io/charts
-  version: 0.7.11
+  version: 0.8.5
```

Comment:
This looks like it should take care of

> The dependency on falco-sidekick has changed, so make sure it is updated too.

Reply:
```suggestion
  version: 0.8.*
```
Do we want a wildcard here instead? https://github.com/elastisys/compliantkubernetes-apps/pull/2280/files#diff-359ae7df31a41985ca7bfbb1b8849bebb9ef68ddcf7d355b44e7dd2c25c1cfd6R49

Reply:
I'm not sure, but do we do such tweaks on upstream charts?